code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -262,6 +264,7 @@
 /datum/reagent/drug/bath_salts
 	name = "Bath Salts"
 	id = "bath_salts"
+	value = 12
```

Comment:
This is only "hard'' because you need to break into the kitchen for materials.

Arguably, meth is harder to make because fucking that up blows up your lab.

Reply:
Yes, you need help from other departments.